src/pages/docs/ai-llms/index.mdx

```diff
@@ -0,0 +1,229 @@
+- You need architectural guidance or best practices
+- Multiple Ably features are involved
+
+Use these to understand **recommended patterns**, not to invent APIs.
+
+#### Canonical reference docs
+Use **canonical docs pages** (API references, SDK references, product docs) when:
+- Providing exact method names, options, defaults, or limits
+- Writing production code
+- Explaining behavior, guarantees, or constraints
+
+Never infer or approximate APIs—always rely on canonical pages.
+
+---
+
+## Product-first documentation priority (very important)
+
+When a product-specific abstraction exists, **always prefer its documentation over generic Pub/Sub docs**, even if the underlying concepts overlap.
+
+Priority order:
+1. **Chat SDK docs** – for chat, rooms, messages, reactions, typing, moderation
+2. **Spaces SDK docs** – for collaborative cursors, avatars, presence in UI
+3. **LiveObjects docs** – for realtime shared state (maps, counters)
+4. **LiveSync docs** – for database-to-client synchronization
+5. **Generic Pub/Sub docs** – only when no higher-level abstraction applies
+
+Do not rebuild Chat, Spaces, or LiveObjects behavior directly on raw channels unless explicitly requested.
+
+---
+
+## Code quality requirements
+
+### Always provide complete code examples
+When you include code:
+- Provide **complete, runnable examples**, not fragments.
+- Include:
+  - imports
+  - client or server initialization
+  - authentication setup (or clear placeholders)
+  - cleanup where relevant
+- Do not ask the user to “fill in the rest” unless they explicitly ask for partial snippets.
+
+### Do not infer or invent code
+- Never invent:
+  - SDK methods
+  - configuration options
+  - defaults or limits
+- Never assume behavior not explicitly documented.
+- If the docs are unclear or incomplete:
+  - say so explicitly
+  - explain what must be verified
+  - ask for clarification if needed
+
+Use placeholders for secrets:
+- `ABLY_API_KEY`
+- `ABLY_AUTH_URL`
+- `ABLY_TOKEN`
+
+---
+
+## Security rules (non-negotiable)
+
+- Never instruct users to embed Ably API keys (with secrets) in browsers or mobile apps.
+- For untrusted clients:
+  - Use **token authentication**
+  - Use a **server-side token issuer**
+  - Scope access using **capabilities**
+- Use `clientId` only when required (presence, identity, moderation).
+- If a proposed approach is insecure, explain why and show the secure alternative.
+
+---
+
+## Choosing the correct Ably interface
+
+Explicitly choose the correct interface and explain why:
+
+- **Realtime SDK**: realtime pub/sub, presence, collaboration
+- **REST SDK**: server-side publishing, token creation, history, stats
+- **Chat SDK**: structured chat features and moderation
+- **Spaces SDK**: collaborative UI primitives
+- **LiveObjects**: realtime shared state
+- **LiveSync**: database-driven realtime updates
+- **SSE / HTTP streaming**: lightweight subscribe-only clients
```

Comment:
I think this will give the wrong impression that if you're only subscribing you should use SSE, that's not the case. SSE etc exist for cases where you can't / don't want to use an Ably SDK.